packages/ui-components/src/components/Panel/Panel.test.tsx

```diff
@@ -9,61 +9,116 @@ import userEvent from "@testing-library/user-event"
 import { describe, expect, test, vi } from "vitest"
 
 import { Panel } from "./Panel.component"
+import { PortalProvider } from "../PortalProvider"
 
 const closedClass = "jn-translate-x-[100%]"
 
 describe("Panel", () => {
   describe("Basic Rendering", () => {
     test("renders a panel", async () => {
-      await waitFor(() => render(<Panel />))
+      await waitFor(() =>
+        render(
+          <PortalProvider>
+            <Panel />
+          </PortalProvider>
+        )
+      )
```

Comment:
[optional]: I'd prefer to extract this to a function like this.
```
const renderPanel = () => {
	return render(
		<PortalProvider>
		  <Panel />
		</PortalProvider>
	)
}
```

Reply:
Sure, let me do this

Reply:
Done